example/apps/example-app/index.js

```diff
@@ -1,49 +1,53 @@
 'use strict';
 
-const controllers = require('hof-controllers');
+const SummaryPageBehaviour = require('hof-behaviour-summary-page');
 
 module.exports = {
   name: 'example-app',
   params: '/:action?',
   steps: {
     '/': {
-      controller: controllers.start,
+      behaviours: require('./behaviours/clear-session'),
```

Comment:
I think it'd be good if we used https://github.com/UKHomeOfficeForms/hof-form-wizard/pull/11 in the example app instead of something custom to achieve the same goal.